### DISCUSSION TITLE
Github support wont answers

### DISCUSSION THREAD

**Discussion by mrvcoder** [community/community#49500] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I submitted a ticket 2 days ago and so far , support is almost 3 days did not answer ! -_-

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The Community is unable to escalate support tickets so we'll be locking this discussion. But please keep in mind that ticket reply time is based on our team's resources and the # of tickets ahead of you. We appreciate your patience!
